### PR TITLE
[Documentation] BackPopulateSNRAndAcquisitionOrder.pl

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Generated using:
pod2markdown tools/BackPopulateSNRAndAcquisitionOrder.pl > docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md

User can type:
perldoc tools/BackPopulateSNRAndAcquisitionOrder.pl 
to get the documentation at the terminal

### CODE REVIEW

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# NAME
+
+BackPopulateSNRAndAcquisitionOrder.pl -- a script that back populates the
+AcqOrderPerModality column of the files table, and the signal-to-noise (SNR)
+values in the parameter\_file table for inserted MINC files. The SNR is computed
+using algorithms built-in the MINC tools.
```
Comment (cmadjar) [inline docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: maybe rephrase "algorithms built-in the MINC tools" to "MINC tools built-in algorithms"?

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# DESCRIPTION
+
+This script will back populate the files table with entries for the
+AcqOrderPerModality column; in reference to:
+https://github.com/aces/Loris-MRI/pull/160
+as well as populate the parameter\_file table with SNR entries in reference to:
+https://github.com/aces/Loris-MRI/pull/142
+It can take in tarchiveID as an argument if only a specific .tar is to be
+processed; otherwise, all .tar in the tarchive table are processed.
```
Comment (cmadjar) [inline docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: replace ".tar" by "DICOM archive (.tar) present"

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# SYNOPSIS
+
+perl tools/BackPopulateSNRAndAcquisitionOrder.pl \`\[options\]\`
+
+Available options are:
+
+\-profile        : name of the config file in
+                  `../dicom-archive/.loris_mri`
+
+\-tarchive\_id    : The tarchive ID of the .tar to be processed from the tarchive
```
Comment (cmadjar) [inline docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: replace ".tar" by "DICOM archive (.tar files)"

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md (hunk 2 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: done

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md (hunk 3 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: done

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md (hunk 1 above)
Comment (MounaSafiHarab) [reply docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: done

File: docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+BACKPOPULATESNRANDACQUIUSsIeTrIOCNoOnRtDrEiRb(u1t)ed PerBlACDKoPcOuPmUeLnAtTaEtSiNoRnANDACQUISITIONORDER(1)
```
Comment (cmadjar) [inline docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: Something funky happened during the creation of the MD. It sttttttutttttttterrreddd ;)

Comment (MounaSafiHarab) [reply docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md]: hahahaha; not sure how this happened! will fix it right away:)

File: tools/BackPopulateSNRAndAcquisitionOrder.pl [hunk 5]
```diff
@@ -1,5 +1,44 @@
...
+                  C<../dicom-archive/.loris_mri>
+
+-tarchive_id    : The tarchive ID of the DICOM archive (.tar files) to be
+                  processed from the C<tarchive> table
+
+
+
+=head1 DESCRIPTION
+
+This script will back populate the files table with entries for the
```
Comment (cmadjar) [inline tools/BackPopulateSNRAndAcquisitionOrder.pl]: files should be in code format

File: tools/BackPopulateSNRAndAcquisitionOrder.pl [hunk 6]
```diff
@@ -1,5 +1,44 @@
...
+AcqOrderPerModality column; in reference to:
```
Comment (cmadjar) [inline tools/BackPopulateSNRAndAcquisitionOrder.pl]: AcqOrderPerModality should be in code format

File: tools/BackPopulateSNRAndAcquisitionOrder.pl [hunk 7]
```diff
@@ -1,5 +1,44 @@
...
+https://github.com/aces/Loris-MRI/pull/160
+as well as populate the parameter_file table with SNR entries in reference to:
```
Comment (cmadjar) [inline tools/BackPopulateSNRAndAcquisitionOrder.pl]: same for parameter_file and SNR

File: tools/BackPopulateSNRAndAcquisitionOrder.pl [hunk 8]
```diff
@@ -1,5 +1,44 @@
...
+https://github.com/aces/Loris-MRI/pull/142
+It can take in tarchiveID as an argument if only a specific DICOM archive
```
Comment (cmadjar) [inline tools/BackPopulateSNRAndAcquisitionOrder.pl]: same for tarchiveID (and t should probably be upper case)

**Comment by MounaSafiHarab** [comment]
> Format table and column names in code please :)

I think I did before I saw your comment, but I did not do it for column names; I will push one more commit now

**Comment by MounaSafiHarab** [comment]
@cmadjar 

done! (I think):)
